resources/common.tf

```diff
@@ -55,9 +53,13 @@ data "digitalocean_ssh_key" "ved-key" {
   name = "Ved SSH Key"
 }
 
+# Sensitive variable. Contains private key for connecting to concrete Digital Ocean droplets
+variable "alexey2_do_private_key" { }
```

Comment:
why do we need to add this private key here?

Reply:
I have done it for connecting cloud TF "null_resourse" to DO droplets.
Private key was saved in cloud TF secure/sensitive variable.

Reply:
I'm not super clear on the architecture. Can you add more details to understand what you thinking ?

My initial thought process was that we want to use a cloud state so that these private/sensitive variables never leave our workspaces. The added benefit with this is that everyone can use their application keys and its much easier for auditing the changes made while the state is kept updated so anyone can pick up with out sharing state explicitly.